src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -95,99 +79,110 @@
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="4"
+                                                Grid.Column="3"
                                                 Content="{x:Bind ColorEntryAt(11), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="5"
+                                                Grid.Column="4"
                                                 Content="{x:Bind ColorEntryAt(12), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="6"
+                                                Grid.Column="5"
                                                 Content="{x:Bind ColorEntryAt(13), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="7"
+                                                Grid.Column="6"
                                                 Content="{x:Bind ColorEntryAt(14), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="8"
+                                                Grid.Column="7"
                                                 Content="{x:Bind ColorEntryAt(15), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
+                                <TextBlock Grid.RowSpan="2"
+                                           Grid.Column="8"
+                                           Padding="10,0,10,0"
+                                           HorizontalAlignment="Center"
+                                           VerticalAlignment="Center"
+                                           AutomationProperties.AccessibilityView="Raw"
+                                           FontFamily="Cascadia Code"

```

Comment:
@DHowett did we ever figure out what happens if we can't find Cascadia Code? Should we care?

Reply:
So, it falls back to Segoe UI or something. It's weird but not world-ending.